src/shared/lib/comparison/ComparisonStore.ts

```diff
@@ -133,7 +133,7 @@ export default abstract class ComparisonStore extends AnalysisStore
     @observable public adjustForLeftTruncation = true;
 
     constructor(
-        protected appStore: AppStore,
+        public appStore: AppStore,
```

Comment:
This one should be made protected again and the featureFlag for the extended survival plot should be shared with <Survival> in a better way.